modules/cluster/variables.tf

```diff
@@ -109,3 +109,9 @@ variable "kms_cmk_arn" {
   default     = ""
   description = "The ARN of the KMS (CMK) customer master key, to be used for Envelope Encryption of Kubernetes secrets, if not set a key will be generated"
 }
+
+variable "nvidia_device_plugin" {
+  type        = bool
+  default     = true
```

Comment:
I set this to default as `true` since it doesn't seem to cause any harm to try and run on a non-gpu node.

But it kindof ties into https://github.com/cookpad/terraform-aws-eks/issues/87#issuecomment-615109193 perhaps we should default stuff like this to false...

Alternatively perhaps we should just add a label to target the DaemonSet to gpu nodes 🤔

Reply:
I went down the nodeSlector route!